src/@types/InteractionContext.d.ts

```diff
@@ -1,4 +1,5 @@
 import type {
+	ButtonInteraction,
 	ChatInputCommandInteraction,
```

Comment:
ESLint tells me that `ButtonInteraction` is never used.

```suggestion
import type {
	ChatInputCommandInteraction,
```

Reply:
fixed